start_cerberus.py

```diff
@@ -328,6 +328,20 @@ def main(cfg):
                     logging.info("Skipping the failed components inspection as "
                                  "it's specific to OpenShift")
 
+                if distribution == "openshift":
+                    watch_csrs_start_time = time.time()
+                    iter_track_time['watch_namespaces'] = time.time() - watch_csrs_start_time
```

Comment:
Why is the time taken for watch_namespaces check being updated here?

Reply:
Copied the wrong item. Took this out now